tifffile/meta.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: tifffile
+  version: 0.3.1
+
+source:
+  fn: tifffile-0.3.1.tar.gz
+  url: https://pypi.python.org/packages/source/t/tifffile/tifffile-0.3.1.tar.gz
+  md5: bd8718457f3578a6016d6dd3c6a041ed
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+# build:
+  #preserve_egg_dir: True
+  #entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - tifffile = tifffile:main
+    #
+    # Would create an entry point called tifffile that calls tifffile.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - numpy
+
+  run:
+    - python
+    - numpy
+
```

Comment:
Strange.  Well, it works for me, and, as far as I can tell, it obeys the conda recipe [package match specification syntax](http://conda.pydata.org/docs/spec.html#package-match-specifications). Are you able to use the recipe I linked to above?  For that matter are you able to use my binary via `conda install -c ilastik tifffile`?